docs/api-reference/type-aliases/MasterEventHandler.mdx

```diff
@@ -0,0 +1,303 @@
+[testplane](../README.mdx) / MasterEventHandler
+
+# Type Alias: MasterEventHandler()\<T\>
+
+```ts
+type MasterEventHandler<T>: (event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T(event, callback) => T;
```

Comment:
Needs fix: Something very weird is going on here. This page should either be deleted or fixed